.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
@asbjornu from my point of view, there is nothing wrong with your solution, but rather, there is something wrong with those files being in there in the first place.  If the `build` folder is intended only for items that are generated as part of the build process, then there shouldn't be any other files/folders in there initially.  This just leads to ambiguity.  For instance, normally in my build script, I will `clean` my artifacts folder, but in this case, I would be deleting files that I shouldn't have, with is not a good place to be.